fleetspeak/client-mac/postinstall

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -e
+
+# Abort if package install is not happening on the running volume.
+[[ $3 != '/' ]] && exit 0
+
+readonly DAEMON_PLIST=/Library/LaunchDaemons/com.google.code.fleetspeak.plist
+
+function clean_up_old_installations() {
```

Comment:
We never released any installers externally that had a version in the package name, so this is not necessary.